packages/bsky/src/hydration/hydrator.ts

```diff
@@ -650,6 +651,19 @@ export class Hydrator {
     const listMemberDids = listsMembers.flatMap((lm) =>
       lm.listitems.map((li) => li.did),
     )
+    const listBlocks: Map<string, Blocks> = new Map()
+    await Promise.all(
+      listUris.map(async (uri) => {
+        const creator = new AtUri(uri).hostname
```

Comment:
Suggest using `didFromUri()` here.

Reply:
(There are a couple more of these lying around but wont leave comments on all of them.)